README.md

```diff
@@ -16,4 +16,4 @@ Feel free to submit pull requests to the game. If merged, they will be shipped i
 - Scenes are in `/Assets/Scenes`. The game's entry-point is `/Assets/Scenes/Bootstrap`
 
 ## SDSH scripts
-Many parts of the game use `sdsh` (Socially Distant SHell) scripts. These are `.sh` files in the bundled assets directory. They use a POSIX-like shell scripting language that the game interprets. When scripting gameplay encounters like missions and NPC interactions, always prefer `sdsh` over C#.
\ No newline at end of file
+Many parts of the game use `sdsh` (Socially Distant Shell) scripts. These are `.sh` files in the bundled assets directory. They use a POSIX-like shell scripting language that the game interprets. When scripting gameplay encounters like missions and NPC interactions, always prefer `sdsh` over C#.

```

Comment:
I believe that one was intended

Reply:
If so, wouldn't **bold** be more appropriate?

Reply:
Bolding it is probably fine

Reply:
Fixed: bde86cac2c2744bb360fe91b32645ea24c23adb0